<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Event::StageWidget</class>
 <widget class="QWidget" name="Event::StageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Date</string>
     </property>
     <property name="buddy">
      <cstring>dateEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="qf::qmlwidgets::DateEdit" name="dateEdit">
     <property name="displayFormat">
      <string>dd.MM. yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&amp;Time</string>
     </property>
     <property name="buddy">
      <cstring>timeEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="qf::qmlwidgets::TimeEdit" name="timeEdit">
     <property name="displayFormat">
      <string>H:mm:ss</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qf::qmlwidgets::TimeEdit</class>
   <extends>QTimeEdit</extends>
   <header location="global">qf/qmlwidgets/timeedit.h</header>
  </customwidget>
  <customwidget>
   <class>qf::qmlwidgets::DateEdit</class>
   <extends>QDateEdit</extends>
   <header location="global">qf/qmlwidgets/dateedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
